Tella/Scenes/Audio/AudioPlayer/ViewModel/EditAudioViewModel.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  EditAudioViewModel.swift
+//  Tella
+//
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Combine
+import Foundation
+import AVFoundation
+import SwiftUI
+
+class EditAudioViewModel: ObservableObject {
+    
+    @Published var startTime: Double = 0.0
+    @Published var endTime: Double = 0.0
+    var shouldReloadVaultFiles : Binding<Bool>?
+    
+    @Published var isPlaying = false
+    var cancellable: Set<AnyCancellable> = []
+    
+    @Published var playingOffset: CGFloat = 0.0
+    
+    var audioUrl: URL?
+    let kNumberOfLabels = 5 // This is for the sub-times labels
+    @Published var currentTime : String  = "00:00:00"
+    @ObservedObject var audioPlayerViewModel: AudioPlayerViewModel
```

Comment:
I suggest to have a new `audioPlayerManager` instance  instead of having  `audioPlayerViewModel` instance in `EditAudioViewModel`